detector/aws_elasticache_cluster_default_parameter_group.go

```diff
@@ -10,20 +10,17 @@ import (
 
 type AwsElastiCacheClusterDefaultParameterGroupDetector struct {
 	*Detector
-	IssueType  string
-	TargetType string
-	Target     string
-	DeepCheck  bool
 }
 
 func (d *Detector) CreateAwsElastiCacheClusterDefaultParameterGroupDetector() *AwsElastiCacheClusterDefaultParameterGroupDetector {
-	return &AwsElastiCacheClusterDefaultParameterGroupDetector{
-		Detector:   d,
-		IssueType:  issue.NOTICE,
-		TargetType: "resource",
-		Target:     "aws_elasticache_cluster",
-		DeepCheck:  false,
-	}
+	nd := &AwsElastiCacheClusterDefaultParameterGroupDetector{Detector: d}
+	nd.Name = "aws_elasticache_cluster_default_parameter_group"
+	nd.IssueType = issue.NOTICE
+	nd.TargetType = "resource"
+	nd.Target = "aws_elasticache_cluster"
```

Comment:
__[gometalinter]__
* [goconst] 6 other occurrence(s) of "aws_elasticache_cluster" found in: aws_elasticache_cluster_duplicate_id.go:23:14 aws_elasticache_cluster_invalid_parameter_group.go:23:14 aws_elasticache_cluster_invalid_subnet_group.go:23:14 aws_elasticache_cluster_invalid_type.go:23:14 aws_elasticache_cluster_previous_type.go:23:14 aws_elasticache_cluster_invalid_security_group.go:24:14

<!-- Comment by [SideCI](https://sideci.com) -->